sparkpost/transmissions.py

```diff
@@ -39,7 +39,13 @@ def _translate_keys(self, **kwargs):
             kwargs.get('use_draft_template', False)
         model['content']['reply_to'] = kwargs.get('reply_to')
         model['content']['subject'] = kwargs.get('subject')
-        model['content']['from'] = kwargs.get('from_email')
+        if kwargs.get('from_name'):
```

Comment:
Sorry about that, I've just removed spaces and did a simple fix to send an empty string if from_name isn't specifyied.
I did the commit. I don't know if I should do another pull request. It's the first time I use github as a contributor :P